lib/file-utilities.js

```diff
@@ -243,7 +243,7 @@ function streamAllLogs(config, callback) {
       for (i = 0; i < data.length; i++) {
```

Comment:
not blocking on this but it looks like we dont need the index anywhere, so could simplify with `for..of` here and in `streamFiles`.

Reply:
Ah, that's part of journald.  I think I'd rather not mess up the scope of this work with those changes because we're deleting that code coming soon.  Lemme know if you feel strongly otherwise.